packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,58 @@
+import omit from "lodash/omit";
+import mongoose from "mongoose";
+import uniqid from "uniqid";
+import { Column } from "../types/Integration";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  id: String,
+  organization: String,
+  table_name: String,
+  columns: [
+    {
+      id: String,
+      column_name: String,
+      data_type: String,
+      path: String,
+    },
+  ],
+  dateCreated: Date,
+  dateUpdated: Date,
+});
+
+type InformationSchemaTablesDocument = mongoose.Document & {
+  id: string;
+  columns: Column[];
+};
```

Comment:
Looks like we're missing the other fields. Normally we use an interface here instead of defining a type inline. Is there a reason we did it this way instead of the way we normally do it? Looks like `table_name`, `organization` and the timestamps are missing.